### DISCUSSION TITLE
Sed task

### DISCUSSION THREAD

**Discussion by Vlek** [wolandark/bash-dungeon#10] [Ideas]
Maybe have a file that needs to be changed so that all instances of something have to be edited so that a chest will open that will allow you to continue?

**Comment by wolandark** [comment]
Good idea for implementing sed/awk. 

It was already decoded that sed and awk tasks will be added.

_Originally posted by @wolandark in https://github.com/wolandark/bash-dungeon/issues/3#issuecomment-2212528504_
